### PR TITLE
[DEV] Pr name check workflow [skip-name]

### PR DESCRIPTION
**Pull request by dmitry-uraev**

### CODE REVIEW

File: target_score.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@
 6
```
Comment (demid5111) [inline target_score.txt]: pipeline should not be checked if the mark is 0 at line 5 (that is score for pipeline) and not if the commit message has the proper mark!

File: config/pr_name_check.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return True
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Checks that PR name is done using the template')
+    parser.add_argument('--pr-name', type=str, help='Current PR name')
+    args: argparse.Namespace = parser.parse_args()
+
+    if '[skip-name]' in args.pr_name:
+        print("Developer Pull Request Passed")
```
Comment (demid5111) [inline config/pr_name_check.py]: `print("Skipping PR name checks.")`

File: .github/workflows/crawler.yml [hunk 3]
```diff
@@ -163,9 +202,21 @@ jobs:
       run: |
         echo "Crawler is working. Proceed to text processing pipeline."
 
+  milestone-2-pipeline:
+    name: Starting pipeline checks!
+    needs: [ milestone-1-crawler-is-working ]
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    if: ${{ ! contains(github.event.head_commit.message, '[skip-pipe]') }}
```
Comment (demid5111) [inline .github/workflows/crawler.yml]: so the condition here should be different

**Review by demid5111 — CHANGES_REQUESTED** [review]

File: target_score.txt (hunk 1 above)
Comment (dmitry-uraev) [reply target_score.txt]: fixed

File: config/pr_name_check.py (hunk 2 above)
Comment (dmitry-uraev) [reply config/pr_name_check.py]: fixed

File: .github/workflows/crawler.yml (hunk 3 above)
Comment (dmitry-uraev) [reply .github/workflows/crawler.yml]: fixed

**Pull request merged**